src/Core/Services/MetadataProviders/Converters/TypeConverter.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Core.Services.MetadataProviders.Converters
+{
+    /// <summary>
+    /// This is a converter to serialize and deserialize an object of Type : System.Type
+    /// For example, the ColumnDefiniton object's default value property of type System.Type.
```

Comment:
Could we add a sample serialized obejct for each converter in description?

Reply:
will add during integration test PR